### ISSUE TITLE
UnityEngine.Rendering gets shown as not being used

### ISSUE THREAD

**Issue by SeniorBriefkasten** [consulo/consulo-unity3d#194]
- IDE build number: 2732
- OS name and version: Windows 10 2004

Unity namespace UnityEngine.Rendering show up as not being used and getting "SerializedDictionary" is not resolved.

![Screenshot 2020-12-13 211811](https://user-images.githubusercontent.com/56098308/102022883-fbd84280-3d89-11eb-9704-cb978c7913eb.png)

**Comment by VISTALL** [comment]
Hello. Can you say SerializedDictionary library? Looks like Consulo don't add this dependency

**Comment by SeniorBriefkasten** [comment]
Hi, I'm not sure if this is right since I'm a beginner but on the [Unity Scripting API site ](https://docs.unity3d.com/Packages/com.unity.render-pipelines.core@7.3/api/UnityEngine.Rendering.SerializedDictionary-2.html)it says Core RP Library

**Comment by VISTALL** [comment]
Just press F12 in Visual Studio (go to definition), and you will see code. If it's decomplied check header

![image](https://user-images.githubusercontent.com/542934/102262147-b5b6e680-3f23-11eb-9ba4-494e62556b03.png)

**Comment by SeniorBriefkasten** [comment]
Hope this is it 

![Screenshot 2020-12-15 215411](https://user-images.githubusercontent.com/56098308/102271999-5f947400-3f20-11eb-9e3d-d92086753da5.png)

**Comment by VISTALL** [comment]
Hello. Looks like it's package 'com.unity.render-pipelines.core' from 'Packages/manifest.json'.

Can you say - directory exists 'Library/PackageCache/com.unity.render-pipelines.core@7.3.0' ?

It exists - try call manual unity sync, right click on root directory - 'Re-import unity project'

**Comment by SeniorBriefkasten** [comment]
I have 'com.unity.render-pipelines.core@**8.3.1'** in my directory.
Re-importing unity project did not help

**Comment by VISTALL** [comment]
Good day. Can you attach Packages/manifest.json?

**Comment by SeniorBriefkasten** [comment]
Hi, here is the manifest hope it helps.

[manifest.txt](https://github.com/consulo/consulo-unity3d/files/5718364/manifest.txt)

**Comment by VISTALL** [comment]
Yeah, it helps. Looks like 'com.unity.render-pipelines.core' is deep depedency (implicit).

For temp fix - add to json file, and call Reimport Unity Project.
